### ISSUE TITLE
The validation subdataset in training

### ISSUE THREAD

**Issue by DetionDX** [davidsandberg/facenet#169]
Hi, David @davidsandberg 

First of all, thanks for your sharing such a excellent work. Also congratulation!!!. One thing I am confusion during reading code, when the CNN is training, your validation data is from the lfw dataset, right? If so, then this is my confusion, should we use the lfw dataset as validation if we test on it? If not, how did you splits your training and validation subdata for training based on CASIA-WebFaces data only?

**Comment by davidsandberg** [comment]
Hi @DetionDX!
Yes, currently LFW is used for validation (and testing) which is obviously quite ugly. I'm not sure how to solve this but one way would be to continue using LFW as a validation set and start using some other dataset for testing, such as the test set in the Megaface challenge or similar.
